React_Fundamentals/src/common/ToDoList/redux/middlewares/fetchTodo.ts

```diff
@@ -0,0 +1,14 @@
+import { Dispatch } from "redux";
+import { addTodo, loadingData } from "../actions";
+
+export const fetchTodo = (text: string) => (dispatch: Dispatch) => {
```

Comment:
I suppose "addTodo" is more appropriate name, because you ADD new item not FETCH it